python/cuml/solvers/cd.pyx

```diff
@@ -305,17 +308,20 @@ class CD(Base):
         y: cuDF DataFrame
```

Comment:
I believe the documentation needs to be update for the output type returned as the output type will be the same as the input provided.

Reply:
That is true in general for the whole codebase, so opened https://github.com/rapidsai/cuml/issues/2243